### PR TITLE
cleared unwanted else clauses and minor readability improvement

### PR DESCRIPTION
**Pull request by navan0**

### CODE REVIEW

**Comment by Navan0** [comment]
@jaingaurav can you approve this?

File: tensorflow/python/module/module.py [hunk 1]
```diff
@@ -106,19 +106,17 @@ def __call__(self, x):
...
+      raise ValueError(
             "%r is not a valid module name. Module names must be valid Python "
             "identifiers (e.g. a valid class name)." % name)
-
+      
     self._name = name
     if tf2.enabled():
       with ops.name_scope_v2(name) as scope_name:
         self._name_scope = ops.name_scope_v2(scope_name)
-    else:
```
Comment (jaingaurav) [inline tensorflow/python/module/module.py]: Don't you still need the else here?

File: tensorflow/python/module/module.py [hunk 2]
```diff
@@ -106,19 +106,17 @@ def __call__(self, x):
   def __init__(self, name=None):
     if name is None:
       name = camel_to_snake(type(self).__name__)
-    else:
-      if not valid_identifier(name):
-        raise ValueError(
+    elif not valid_identifier(name):
+      raise ValueError(
             "%r is not a valid module name. Module names must be valid Python "
```
Comment (jaingaurav) [inline tensorflow/python/module/module.py]: Shouldn't the indentation be adjusted here?

File: tensorflow/python/module/module.py [hunk 3]
```diff
@@ -355,8 +352,7 @@ def _flatten_module(module,
       if predicate(leaf):
         if with_path:
           yield module_path + leaf_path, leaf
-        else:
```
Comment (jaingaurav) [inline tensorflow/python/module/module.py]: I'm not sure I agree that removing this else improves readability. I'd recommend adding it back.

**Review by jaingaurav — CHANGES_REQUESTED** [review]

**Pull request closed without merging**